eventmesh-runtime/build.gradle

```diff
@@ -23,14 +23,21 @@ List metrics = [
         "io.dropwizard.metrics:metrics-json:4.1.0"
 ]
 
-
+List open_telemetry = [
+        "io.opentelemetry:opentelemetry-api:1.3.0",
+        "io.opentelemetry:opentelemetry-sdk:1.3.0",
+        "io.opentelemetry:opentelemetry-sdk-metrics:1.3.0-alpha",
```

Comment:
Can we use other stable version? Using alpha does not seem to be a good choice.

Reply:
It is ok to use that. Otel's sdk metrics is in alpha although, it is widely used in https://github.com/open-telemetry/opentelemetry-java-instrumentation. This agent has been validated in enough prod environments.

Reply:
OK